packages/api/cms-api/src/index.ts

```diff
@@ -73,6 +72,7 @@ export { defaultDamAcceptedMimetypes } from "./dam/common/mimeTypes/default-dam-
 export { DamConfig } from "./dam/dam.config";
 export { DAM_CONFIG, IMGPROXY_CONFIG } from "./dam/dam.constants";
 export { DamModule } from "./dam/dam.module";
+export { EntityInfo, EntityInfoGetter, EntityInfoServiceInterface, GetEntityInfo } from "./dam/files/decorators/entity-info.decorator";
```

Comment:
I'd only export EntityInfo and EntityInfoServiceInterface. Also EntityInfoGetter and GetEntityInfo are not easily distinguishable

Reply:
Yes the other exports aren't really necessary.

How would you name  `EntityInfoGetter` and `GetEntityInfo`?